rebasehelper/tests/functional/test_rebase.py

```diff
@@ -56,6 +56,7 @@ class TestRebase:
         'rebase/applicable.patch',
         'rebase/conflicting.patch',
         'rebase/backported.patch',
+        'rebase/0.1.patch',
```

Comment:
Looking at the list, I think it would be better to order the patches alphabetically, as that's how they are ordered in the diff.